fieldservice_equipment_stock/__manifest__.py

```diff
@@ -10,9 +10,7 @@
     "Brian McMaster, "
     "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/field-service",
-    "depends": [
-        "fieldservice_stock",
-    ],
+    "depends": ["fieldservice_stock", "fieldservice_stock_request"],
```

Comment:
This is not reasonable, is it force you to use additional modules you might not be interested in.